tools/src/test/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/YugipediaPrintMapperTest.java

```diff
@@ -0,0 +1,53 @@
+package io.github.ygojson.tools.dataprovider.impl.yugipedia.mapper;
+
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import org.approvaltests.Approvals;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mapstruct.factory.Mappers;
+
+import io.github.ygojson.model.data.Print;
+import io.github.ygojson.model.utils.JsonUtils;
+import io.github.ygojson.tools.dataprovider.impl.yugipedia.YugipediaTestData;
+import io.github.ygojson.tools.dataprovider.impl.yugipedia.model.CardTable2;
+
+class YugipediaPrintMapperTest {
```

Comment:
Move to the new structure once https://github.com/ygojson/ygojson-tools/pull/60 is ready